src/modules/Dashboard/features/Applet/Popups/AddParticipantPopup/AddParticipantPopup.tsx

```diff
@@ -179,9 +179,6 @@ export const AddParticipantPopup = ({
           title={t('addParticipant')}
           buttonText={t('next')}
           hasLeftBtn
-          leftBtnText={t('addViaCSV')}
-          leftBtnVariant="outlined"
-          onLeftBtnSubmit={handleAddViaCSV}
```

Comment:
Could you also delete the definition of `handleAddViaCSV` earlier in this file? Its presence has introduced a new eslint warning after this change.

Reply:
Good catch, we have so many warnings that its difficult to spot when they've changed 😅.